From 913b2f1265bd7f3ebe6cf4cba92c20c5a423a6c3 Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Tue, 2 Aug 2016 12:05:55 +0200
Subject: [PATCH 5/6] Don't include missing and unnecessary math header

This removes the inclusion of an unnecessary math header that is neither
found nor necessary for non-cortex-m ECs.

Change-Id: I56a04178dadedb76f841504fa645e2d43900d25f
Signed-off-by: Paul Kocialkowski <contact@paulk.fr>
---
 common/math_util.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/common/math_util.c b/common/math_util.c
index 120d13d..34a267d 100644
--- a/common/math_util.c
+++ b/common/math_util.c
@@ -6,7 +6,6 @@
 /* Common math functions. */
 
 #include "common.h"
-#include "math.h"
 #include "math_util.h"
 #include "util.h"
 
-- 
2.9.0

